client/solve.go

```diff
@@ -326,6 +328,9 @@ func (c *Client) solve(ctx context.Context, def *llb.Definition, runGateway runG
 				if err == io.EOF {
 					return nil
 				}
+				if status.Code(errors.Cause(err)) == codes.Canceled {
```

Comment:
can you add a comment such as `// earthly-specific: ...` to make it more clear this was added by us. This makes it much easier to deal with upstream merges.

Reply:
Done